### DISCUSSION TITLE
Welcome to SPlayer Discussions!

### DISCUSSION THREAD

**Discussion by imsyy** [SPlayer-Dev/SPlayer#7] [公告 Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by losie** [comment]
加油！

**Reply by imsyy** [reply]
感谢支持 🎉

**Comment by Rex-noir** [comment]
Bro is there any english version of splayer for linux? I freaking love the UI but I can't read anything.

**Comment by sandianshui123** [comment]
大佬，能不能在本地音乐，加一个最近新加歌曲 的分类排序
